rest/route/test.go

```diff
@@ -119,25 +131,34 @@ func (tgh *testGetHandler) Run(ctx context.Context) gimlet.Responder {
 		if startAt*tgh.limit < filteredCount {
 			key = fmt.Sprintf("%d", startAt+1)
 		}
-	} else {
+	} else if err != nil {
+		// No cedar test results, fall back to the database.
 		grip.Warning(message.WrapError(err, message.Fields{
 			"task_id": tgh.taskID,
 			"message": "problem getting cedar test results",
 		}))
 
-		// No cedar test results, fall back to the database.
-		tests, err = tgh.sc.FindTestsByTaskId(tgh.taskID, tgh.key, tgh.testName, tgh.testStatus, tgh.limit+1, tgh.testExecution)
-		if err != nil {
-			return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "database error"))
-		}
+		if tgh.testID != "" {
```

Comment:
This case seems kind of weird because it only uses the test ID and completely disregards the task ID from the route. I don't care that much about it, but it is pretty weird for a route to require a task ID, then ignore the task ID.

I'm okay with this as long as we get rid of this code eventually and switch entirely to Cedar test results.

Reply:
Yeah it is weird, I agree. I kept the route though because to do this using cedar we need the taskid and execution. This will be legacy though.